action.yml

```diff
@@ -5,9 +5,12 @@ branding:
   color: 'purple'
 description: Comment on pull requests
 inputs:
-  domain:
-    template: template file name
+  template:
+    description: template file name
```

Comment:
these arguments I think were not correct. I've updated everywhere to be `template`.
Let me know if this is incorrect

Reply:
The build has failed expecting `domain` but the code previously was looking for `template` is why I updated to `template`